integration_test/src/test_runner/test_generator_util.rs

```diff
@@ -88,6 +88,7 @@ pub fn generate_baseline_test_file(entry: &TestEntry, stmt: HStmt) -> Result<()>
     let test_entry = TestEntry {
         description: entry.description.clone(),
         db: entry.db.clone(),
+        simple_type: entry.simple_type,
```

Comment:
[nit] For flags, use the form "is_xxx". It makes it very easy to identify the purpose this way.